eng/Versions.props

```diff
@@ -6,10 +6,10 @@
     <PreReleaseVersionLabel>beta</PreReleaseVersionLabel>
     <!-- Opt-out repo features -->
     <UsingToolXliff>false</UsingToolXliff>
-    <MicrosoftNETTestSdkVersion>17.0.0</MicrosoftNETTestSdkVersion>
+    <MicrosoftNETTestSdkVersion>17.1.0-preview-20211118-01</MicrosoftNETTestSdkVersion>
```

Comment:
This version will be overridden by Arcade,

Reply:
See: https://github.com/microsoft/testfx/pull/1007/files#diff-1ea18ff65faa2ae6fed570b83747086d0317f5e4bc325064f6c14319a9c4ff67s :)

Reply:
Better?

Reply:
I edited the file manually, now it should give warning about downgrade if Maestro tries to patch over it.